genome_designer/genome_finish/insertion_placement_read_trkg.py

```diff
@@ -472,11 +474,20 @@ def write_read_query_alignments_to_fastq(reads, fastq_path,
 
 def simple_align_with_bwa_mem(reads_fq, reference_fasta, output_bam_path):
 
+    if not os.path.exists(settings.TEMP_FILE_ROOT):
+            os.mkdir(settings.TEMP_FILE_ROOT)
+
+    filename_prefix = 'temp_ref_copy'
```

Comment:
What exactly is the bug? Ideally we would not generate copies of the reference genome.

At the least I would expect you to have a randomly generated name here (uuid). And then somewhere this temp reference should be deleted.